simulator/IsSimulatorPlugin.kt

```diff
@@ -0,0 +1,8 @@
+package venusbackend.simulator
+
+import venusbackend.riscv.MachineCode
+
+interface IsSimulatorPlugin {
```

Comment:
Why is this not just titled SimulatorPlugin.

Reply:
will change